examples/querylog1

```diff
@@ -0,0 +1 @@
tuple(5)) WHERE equals(deleted, 0) AND greaterOrEquals(_snuba_timestamp, toDateTime('2023-02-08T21:07:12', 'Universal')) AND less(_snuba_timestamp, toDateTime('2023-02-08T21:12:39', 'Universal')) AND in((project_id AS _snuba_project_id), tuple(1)) AND in(_snuba_project_id, tuple(1)) GROUP BY _snuba_tags_key, _snuba_tags_value ORDER BY _snuba_count DESC LIMIT 4 BY _snuba_tags_key LIMIT 1000 OFFSET 0", "sql_anonymized": "SELECT (tupleElement((arrayJoin(arrayMap((x, y -> (x, y)), tags.key, tags.value)) AS snuba_all_tags), -1337) AS _snuba_tags_key), (tupleElement(snuba_all_tags, -1337) AS _snuba_tags_value), (count() AS _snuba_count), (min((timestamp AS _snuba_timestamp)) AS _snuba_first_seen), (max(_snuba_timestamp) AS _snuba_last_seen) FROM errors_local PREWHERE in((group_id AS _snuba_group_id), tuple(-1337)) WHERE equals(deleted, -1337) AND greaterOrEquals(_snuba_timestamp, toDateTime('2023-02-08T21:07:12', 'Universal')) AND less(_snuba_timestamp, toDateTime('2023-02-08T21:12:39', 'Universal')) AND in((project_id AS _snuba_project_id), tuple(-1337)) AND in(_snuba_project_id, tuple(-1337)) GROUP BY _snuba_tags_key, _snuba_tags_value ORDER BY _snuba_count DESC LIMIT 4 BY _snuba_tags_key LIMIT 1000 OFFSET 0", "start_timestamp": 1675919232, "end_timestamp": 1675919559, "stats": {"clickhouse_table": "errors_local", "final": false, "referrer": "tagstore.__get_tag_keys_and_top_values", "sample": null, "table_rate": 0.6, "table_concurrent": 1, "project_rate": 0.6333333333333333, "project_concurrent": 1, "consistent": false, "result_rows": 22, "result_cols": 5, "query_id": "9079915acbacff0804ed45c72b865024"}, "status": "success", "trace_id": "8377f280e7eb4754a7f20df73ce2cf37", "profile": {"time_range": null, "table": "errors_local", "all_columns": ["errors_local.deleted", "errors_local.group_id", "errors_local.project_id", "errors_local.tags.key", "errors_local.tags.value", "errors_local.timestamp"], "multi_level_condition": false, "where_profile": {"columns": ["errors_local.deleted", "errors_local.project_id", "errors_local.timestamp"], "mapping_cols": []}, "groupby_cols": ["errors_local.tags.key", "errors_local.tags.value"], "array_join_cols": ["errors_local.tags.key", "errors_local.tags.value"]}, "result_profile": {"bytes": 1305, "blocks": 1, "rows": 22, "elapsed": 0.009863138198852539}}], "status": "success", "timing": {"timestamp": 1675890758, "duration_ms": 55, "marks_ms": {"cache_get": 2, "cache_set": 6, "execute": 10, "get_configs": 0, "prepare_query": 15, "rate_limit": 5, "validate_schema": 15}, "tags": {}}, "projects": [1], "snql_anonymized": "MATCH Entity(events) SELECT tags_key, tags_value, (count() AS count), (min(timestamp) AS first_seen), (max(timestamp) AS last_seen) GROUP BY tags_key, tags_value WHERE greaterOrEquals(timestamp, toDateTime('$S')) AND less(timestamp, toDateTime('$S')) AND in(project_id, tuple(-1337)) AND in(project_id, tuple(-1337)) AND in(group_id, tuple(-1337)) ORDER BY count DESC LIMIT 4 BY tags_key LIMIT 1000 OFFSET 0"}
```

Comment:
the example does not pass the schema. I tested with https://www.jsonschemavalidator.net/

I would add CI for validating all examples (probably via pytest)

Reply:
Oops, thanks! Will tackle the CI in a follow up.